src/assets/scss/styles.scss

```diff
@@ -1,2 +1,9 @@
-/*.active {
```

Comment:
this css is for the navbar, to be reused later.

Reply:
Shall we not put it this into the nav css instead ? This code was commented without any specification.

Reply:
This code is moved to nav-specific css for futur use